src/doubleLinkedList.js

```diff
@@ -0,0 +1,150 @@
+'use strict'
+
+class Node {
+  constructor(data) {
+    this.data = data
+    this.next = null
+    this.prev = null
+  }
+}
+
+export default class DoubleLinkedList {
+  constructor() {
+    this.count = 0
+    this.head = null
+    this.tail = null
+  }
+
+  getHeadNode() {
+    return this.head
+  }
+
+  getTailNode() {
+    return this.tail
+  }
+
+
+  find(item) {
+    let currentNode = this.head
+      while (currentNode) {
```

Comment:
Indentation should be consistently left aligned at the same level of a given block.